components/DropdownMenu.vue

```diff
@@ -56,14 +56,14 @@ function menuItemClick(event: Event, item: DropdownMenuItem) {
         </p>
         <p
           v-else
-          @click="(event: MouseEvent) => event.stopPropagation()"
           class="flex justify-start items-center gap-1 py-2 px-3"
+          @click="(event: MouseEvent) => event.stopPropagation()"
```

Comment:
What is the `stopPropagation` for?

Reply:
I don't remember that i made changes in this file.

Reply:
I think it is @bjornarhagen who worked on that !!